app/src/main/java/com/onegini/mobile/exampleapp/OneginiConfigModel.java

```diff
@@ -26,8 +26,8 @@
   private final String appPlatform = "android";
   private final String redirectionUri = "oneginiexample://loginsuccess";
   private final String appVersion = "2.2.0";
-  private final String baseURL = "https://demo-msp.onegini.com";
-  private final String resourceBaseURL = "https://demo-msp.onegini.com/resources";
+  private final String baseURL = "https://onegini-msp-snapshot.test.onegini.io";
+  private final String resourceBaseURL = "https://onegini-msp-snapshot.test.onegini.io/resources";
```

Comment:
We can always revert the commit. We change those every time we want to develop or tests something.